Sources/LiveKit/Protocols/ParticipantDelegate.swift

```diff
@@ -79,6 +79,10 @@ public protocol ParticipantDelegate: AnyObject {
     @objc(localParticipant:didUnpublishTrack:) optional
     func participant(_ participant: LocalParticipant, didUnpublishTrack publication: LocalTrackPublication)
 
+    /// Fired when the first remote participant has subscribed to the localParticipant's track.
+    @objc(localParticipant:didSubscribeTrack:) optional
+    func participant(_ participant: LocalParticipant, didSubscribeTrack publication: LocalTrackPublication)
```

Comment:
This name sounds like the local participant subscribed to the track, wonder if i can put a better name for this.